CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 2.1.1 (Next)
```

Comment:
Let's bump the version to 2.2 and make a release?

Reply:
Makes sense. I'm going to put up another PR to replace some of the patched argument validations with the new `arg_groups` data from the ref, so I'll do it in that.